coderedcms/blocks/__init__.py

```diff
@@ -82,6 +83,10 @@
 ]
 
 CONTENT_STREAMBLOCKS = HTML_STREAMBLOCKS + [
+    ('accordion', AccordionBlock([
+        ('row', GridBlock(HTML_STREAMBLOCKS)),
```

Comment:
Do we really want grids inside of accordions? I think this is adding way too many levels. Previously our max possible was 5 levels deep. This increases it to 7.

HeroBlock > GridBlock > ColumnBlock > AccordionBlock > GridBlock > ColumnBlock > Text.